dbos/_sys_db.py

```diff
@@ -227,26 +225,26 @@ class StepInfo(TypedDict):
 
 class SystemDatabase:
 
-    def __init__(self, database: DatabaseConfig, *, debug_mode: bool = False):
-        sysdb_name = (
-            database["sys_db_name"]
-            if "sys_db_name" in database and database["sys_db_name"]
-            else database["app_db_name"] + SystemSchema.sysdb_suffix
-        )
+    def __init__(
+        self,
+        *,
+        database_url: str,
+        engine_kwargs: Dict[str, Any],
+        sys_db_name: Optional[str] = None,
```

Comment:
Future PR: I'll make this mandatory and solved upstream by all consumers of the class (e.g., client)